src/app/Components/LanguageSources.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+  export let sources: any[];
+</script>
+
+<style>
+  .language__sources {
+    border-radius: 20px;
+    padding: 5px;
+  }
+
+  th {
+    text-align: left;
+    background-color: #b6b6b6;
+    padding: 20px 20px 20px 5px;
+  }
+
+  table {
+    border-collapse: collapse;
+    width: 1400px;
+  }
+
+  tr {
+    border-bottom: 1pt solid black;
+  }
+
+  .table__row--actions {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .table__row--actions p {
+    margin-left: 10px;
+  }
+</style>
+
+<div class="language__sources">
+  <table>
+    <tr>
+      <th>Name</th>
+      <th>Size</th>
+      <th>Type</th>
+      <th>Actions</th>
+    </tr>
```

Comment:
Excellent use of `<th>`! I would also  use the [`<thead>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) element here.

```suggestion
    <thead>
    <tr>
        <th>Name</th>
        <th>Size</th>
        <th>Type</th>
        <th>Actions</th>
      </tr>
    </thead>
```